projects/OpenWebCalendar/default.nix

```diff
@@ -50,8 +49,7 @@
       description = "Deployment for demo purposes";
       # FIX: re-enable after this is solved:
       # https://github.com/NixOS/nixpkgs/issues/418689
-      # tests.basic = "${sources.inputs.nixpkgs}/nixos/tests/web-apps/open-web-calendar.nix";
-      tests.basic = null;
+      # tests.basic = pkgs.nixosTests.open-web-calendar;
```

Comment:
woulnd't we apply the `broken` annotation here?

Reply:
We can do that after https://github.com/ngi-nix/ngipkgs/pull/1257. I took this change from that PR to narrow down the CI failure scope (which we only need to disable the Pixelfed test in aarch64, since it's [not supported](https://github.com/NixOS/nixpkgs/blob/53b8a4618934db6cbcc02359ccfd49ef034f782a/nixos/tests/web-apps/pixelfed/default.nix#L6-L9)).